### PR TITLE
Move build_description setting to job stage due to different pipeline…

### PR DESCRIPTION
**Pull request by ehelms**
… agents

### CODE REVIEW

**Comment by ehelms** [comment]
Testing this first

File: puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/lib/foreman_infra.groovy [hunk 1]
```diff
@@ -15,5 +15,5 @@ def set_job_build_description() {
        build_description += "<a href=\"${link}\">${files_list[i]}</a><br/>"
     }
 
-    currentBuild.description = build_description
+    currentBuild.description += build_description
```
Comment (ekohl) [inline puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/lib/foreman_infra.groovy]: I wonder if description is ever null

Comment (ehelms) [reply puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/lib/foreman_infra.groovy]: You wonder correctly, and I had been testing an update to handle that.

**Comment by ehelms** [comment]
@ekohl check the second commit to handle the root ownership issues we've been seeing

**Pull request merged**